templates/components/streamfield/blocks/image_block.html

```diff
@@ -1,5 +1,12 @@
 {% verbatim %}
-<div class="">
-    {{ self }}
+{% load wagtailimages_tags %}
+
+<div class="pb-10 md:pb-20 flex justify-center">
+  <div class="mx-auto">
+    {% image value.image width-640 class="mx-auto" %}
```

Comment:
From my inspection it seems the `mx-auto` is not necessary in this case because the `justify-center` flex wrapper will automatically center-align the child div if it doesn't take up the whole space.

```suggestion
  <div>
    {% image value.image width-640 %}
```